contrib/camel/samples/source_telegram.yaml

```diff
@@ -0,0 +1,24 @@
+# Apache Camel Telegram Source
```

Comment:
This doesn't seem to be documented in the README.md?

Reply:
Uh, you're right.. :sweat_smile: